chapters/en/slides/module8/module8_03.md

```diff
@@ -0,0 +1,367 @@
+
+```{math}
+:label: eq_label
+m_{j} \approx 1.9 \times 10^{27} \: \text{kg}
+```
+
+Equation {eq}`eq_label` is an example of a named equation.
+~~~
+
+Notes:
+
+If you have created an equation with a label, you can link to it from within your text and across pages.
+
+You can refer to the equation using the label that you’ve provided by using `` {eq}`eq_label` ``, just like ``{numref}`jupiter-figure` `` that we used with images.
+
+<br>
+
+Here is how we add equations to `mymarkdownfile.md`:
+
+---
+
+# Cross-referencing equations
+
+Once you add these equations to `mymarkdownfile.md` and rebuild your Jupyter Book, your book will look like:
+
+<img style="border:1px solid black;" src="/module8/equation.png" width="750"></img>
+
+Notes:
+
+Once you add these equations to `mymarkdownfile.md` and rebuild your Jupyter Book, your book will look like:
+
+<img style="border:1px solid black;" src="/module8/equation.png" width="750"></img>
+
+---
+
+# Margin content
+
+Sometimes it is helpful to add notes to the margin of a page. This is similar to _footnotes_ in a conventional book.
+
+In Jupyter Book, you can do that using `{margin}` blocks in a markdown file:
+
+~~~
+```{margin} Did you know?
+Jupiter is 11.0x larger than Earth!
+```
+~~~
+
+<br>
+
+Here is how it appears in our Jupyter Book:
+
+<img style="border:1px solid black;" src="/module8/margin.png" width="800"></img>
+
+Notes:
+
+Sometimes it is helpful to add notes to the margin of a page. This is similar to _footnotes_ in a conventional book.
+
+In Jupyter Book, you can do that using `{margin}` blocks in a markdown file:
+
+~~~
+```{margin} Did you know?
+Jupiter is 11.0x larger than Earth!
+```
+~~~
+
+<br>
+
+Here is how it appears in our Jupyter Book:
+
+<img style="border:1px solid black;" src="/module8/margin.png" width="800"></img>
+
+---
+
+# Admonitions
+
+Admonitions in Jupyter Book are visual elements that you can use to draw attention to important information.
+
+There are different types of admonitions that you can use in Jupyter Book, including _Note_, _Tip_, _Warning_, _Danger_ and _Hint_.
+
+You can define an admonition by inserting an admonition code block:
+
+~~~
+```{note}
+I'm a note!
+```
+~~~
+
+`{note}` can be replaced with `{warning}`, `{tip}`, `{danger}` and so on. This is how these admonitions will show up:
+
+<img src="/module8/admonition.png" width="600"></img>
+
+Notes:
+
+Admonitions in Jupyter Book are visual elements that you can use to draw attention to important information.
+
+There are different types of admonitions that you can use in Jupyter Book, including _Note_, _Tip_, _Warning_, _Danger_ and _Hint_.
+
+You can define an admonition by inserting an admonition code block:
+
+~~~
+```{note}
+I'm a note!
+```
+~~~
+
+`{note}` can be replaced with `{warning}`, `{tip}`, `{danger}` and so on. This is how these admonitions will show up:
+
+<img src="/module8/admonition.png" width="600"></img>
+
+---
```

Comment:
Maybe we can add some "cool" element, like a drop down or panel, so that they see a bit more of the interactive components? https://jupyterbook.org/en/stable/content/content-blocks.html#dropdowns

Reply:
Sure, I added slides for panels, simple dropdowns and dropdowns with admonition.